cmd/builder/internal/builder/config.go

```diff
@@ -212,6 +214,51 @@ func (c *Config) allComponents() []Module {
 	return slices.Concat[[]Module](c.Exporters, c.Receivers, c.Processors, c.Extensions, c.Connectors, c.ConfmapProviders, c.ConfmapConverters)
 }
 
+func (c *Config) getGoBuildArgs() []string {
+	ldflags := "-s -w" // we strip the symbols by default for smaller binaries
+	gcflags := ""
+	executableName := c.Distribution.Name
+	if runtime.GOOS == "windows" {
+		executableName += ".exe"
```

Comment:
Codecov seems to think these aren't covered, but I think they are covered by `TestGetGoBuildArgs`.

Reply:
the reason you're getting this warning is because the body of the if statement isn't being run in codecov because it's not running in windows. I think this finding should probably be ignored because the test TestGetGoBuildArgs explicitly tests correctly on windows systems